manifests/10_deployment.yaml

```diff
@@ -35,10 +35,7 @@ spec:
       priorityClassName: system-cluster-critical
       serviceAccountName: cluster-storage-operator
       securityContext:
-        fsGroup: 10400
-        runAsGroup: 10400
         runAsNonRoot: true
-        runAsUser: 10400
         seccompProfile:
```

Comment:
CSO seems to be running in `nonroot-v2` SCC , which has following policies:

```
fsGroup:
  type: RunAsAny
```

Have you tested if this still results in assigning a fsgroup to the pod?

Reply:
Yes, CI shows that CSO runs with some fsGroup:

https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-storage-operator/316/pull-ci-openshift-cluster-storage-operator-master-e2e-azure-csi/1570712022071382016

https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_cluster-storage-operator/316/pull-ci-openshift-cluster-storage-operator-master-e2e-azure-csi/1570712022071382016/artifacts/e2e-azure-csi/gather-extra/artifacts/pods.json

```
                "name": "cluster-storage-operator-6db7799b89-rxmhq",
                "namespace": "openshift-cluster-storage-operator",
...

                "securityContext": {
                    "fsGroup": 1000150000,
                    "runAsNonRoot": true,
                    "seLinuxOptions": {
                        "level": "s0:c12,c9"
                    },
                    "seccompProfile": {
                        "type": "RuntimeDefault"
                    }
                },
                "serviceAccount": "cluster-storage-operator",
                "serviceAccountName": "cluster-storage-operator",
```

Reply:
BTW, SCC changed to:

```
                    "openshift.io/scc": "restricted-v2",
```